pages/common/duplicacy.md

```diff
@@ -0,0 +1,36 @@
+# duplicacy
+
+> A lock-free deduplication cloud backup tool.
+> More information: <https://github.com/gilbertchen/duplicacy/blob/master/GUIDE.md>.
+
+- Use current directory as the repository, initialize a SFTP storage and encrypt the storage with a password:
+
+`duplicacy init -e {{repository_id}} {{sftp://user@192.168.2.100/path/to/storage/}}`
```

Comment:
https://github.com/gilbertchen/duplicacy/wiki/init says this positional argument is `snapshot id`. I'm not sure if it is the same as `repository id`.
```suggestion
`duplicacy init -e {{snapshot_id}} {{sftp://user@192.168.2.100/path/to/storage/}}`
```

Reply:
Yes, you're right! I always misuse the repository_id and snapshot_id here. Thank you for point out!